ext/gosu/gosu.i

```diff
@@ -1014,6 +1014,12 @@ namespace Gosu
     {
         $self->Gosu::Window::close();
     }
+
+    Gosu::Image to_image()
```

Comment:
Huh, does that work? I think I'm returning `Gosu::Image*` in other places. Would be nice if it could all use the pointerless variant

Reply:
Well - I got no errors, so I guess it works?